scripts/cozy-app-dev.sh

```diff
@@ -104,6 +104,7 @@ do_start() {
 
 	${COZY_STACK_PATH} serve --allow-root \
 		--appdir "${appdir}" \
+		--host "::" \
```

Comment:
I'd have put `0.0.0.0` instead of `::`, but it should work too

Reply:
IPv6 soon ? :joy: